cs/src/Contracts/TunnelConstraints.cs

```diff
@@ -137,7 +137,7 @@ public static class TunnelConstraints
     /// Cluster IDs are alphanumeric; hyphens are not permitted.
     /// </remarks>
     /// <seealso cref="Tunnel.ClusterId"/>
-    public const string ClusterIdPattern = "[a-z][a-z0-9]{2,11}";
+    public const string ClusterIdPattern = "^(([a-z]{3,4}[0-9]{1,3})|asse|aue|brs|euw|use)$";
```

Comment:
Should we include "localhost1" and "localhost2" cluster IDs used by testing?

Reply:
Discussed with Ilya, the solution for us is to use valid cluster IDs instead of `localhost1` and `localhost2` in our testing environments.